packages/component-base/src/uri-utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Check if a path matches to any within an array of paths
+ *
+ * @param {string} pathToMatch
+ * @param {Array<String>} paths
+ */
+export function isMatchingPath(pathToMatch, paths) {
```

Comment:
After more consideration, I think it's better to keep the helper as simple as possible, that is, don't make it accept an array of paths.  

```suggestion
export function isMatchingPath(path1, path2) {
```

Reply:
Done.